### DISCUSSION TITLE
Tailwind Problem

### DISCUSSION THREAD

**Discussion by Marvin678** [neuefische/questions#716] [Web] [answered]
Moin,

ich habe versucht Tailwind einzurichten und immer wenn ich den Befehl ` npx tailwindcss init ` eingebe kommt folgende Fehlermeldung `sh: tailwindcss: command not found`.

Hoffe jemand kann helfen.

**Comment by Kheirah** [comment, marked as answer]
I would need more specifics to properly address the problem. Are you using React? Are using Next? How did you set up the project?

Generally speaking, you can always try deleting the `node_modules` directory and see it if helps.

**Reply by Marvin678** [reply]
deleting the `node_modules` helped me. Thx
